docs/developer-docs/integrations/sns/managing/making-proposals.md

```diff
@@ -0,0 +1,98 @@
+# SNS proposals
+
+## Overview
+
+To manage an SNS, the SNS community needs to understand how proposals work, how they can be submitted, voted on, and what effect they have. This article explains the different types of proposals and how to work with them.
+
+## Background
+
+### Native proposals
+
+An SNS comes with built-in proposals called “native proposals”.
+
+There are the following types:
+
+* `motion`
+* `manage_nervous_system_parameters`
+* `upgrade_sns_controlled_canister`
+* `add_generic_nervous_system_function`
+* `remove_generic_nervous_system_function`
+* `execute_generic_nervous_system_function`
+* `upgrade_sns_to_next_version`
+* `manage_sns_metadata`
+* `transfer_sns_treasury_funds`
+* `register_dapp_canisters`
+* `deregister_dapp_canisters`
+
+:::info
+See the types in the code [here](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto?L405) - they are called “action” in the code.
```

Comment:
```suggestion
See the types in the code [here](https://sourcegraph.com/github.com/dfinity/ic@4732d8281404c0a7c1e0a91937ffd0e54f2beced/-/blob/rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto?L405) - they are called “action” in the code.
```
I recommend making this a permalink to mitigate weird linkage problems with the tip of master